### PR TITLE
Fix bug

### PR DESCRIPTION
**Pull request by DrAnax**

### CODE REVIEW

**Comment by Zamiell** [comment]
SLR

**Comment by Zamiell** [comment]
meaning break this up into multiple lines because that is too much for one line to do

**Comment by DrAnax** [comment]
> 

Read it as "Sorry late reply", so I thought you were busy playing and went to sleep.

**Comment by Zamiell** [comment]
no, SLR is my favorite acronym

https://midu.dev/single-line-responsability-haz-una-cosa-por-linea/

File: packages/client/src/lobby/createReplayJSON.ts [hunk 1]
```diff
@@ -207,7 +207,8 @@ function getCardFromHypoState(
   if (globals.state.replay.hypothetical === null) {
     return 0;
   }
-  return globals.state.replay.hypothetical.states[stateIndex].hands[
+  const minStateIndex = Math.max(stateIndex, 0);
+  return globals.state.replay.hypothetical.states[minStateIndex].hands[
```
Comment (Zamiell) [inline packages/client/src/lobby/createReplayJSON.ts:211]: we can do better than this.
the line is still doing more than one thing.
keep breaking it up with good variable names that describe what is actually happening

**Comment by DrAnax** [comment]
Why tests do not run?

**Comment by Zamiell** [comment]
you can do even more SLR here

**Comment by Zamiell** [comment]
how about this:
```ts
const stateOnHypoTurn = globals.state.replay.hypothetical.states[minStateIndex]; // is this the right variable name? what specific state is this?
const playerHand = state.hands[playerIndex];
return playerHand [slotIndex];
```

**Comment by Zamiell** [comment]
much better thanks :)

**Pull request merged**